packages/angular_devkit/build_angular/src/babel/webpack-loader.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { custom } from 'babel-loader';
+
+interface AngularCustomOptions {
+  forceES5: boolean;
+  requiresLinking: boolean;
+}
+
+export default custom<AngularCustomOptions>(() => {
+  const baseOptions = Object.freeze({
+    babelrc: false,
+    configFile: false,
+    compact: false,
+    cacheCompression: false,
+    sourceType: 'unambiguous',
+  });
+  let hasLinkerSupport = false;
+
+  return {
+    customOptions({ forceES5, ...loaderOptions }, { source }) {
+      let requiresLinking = false;
+      try {
+        requiresLinking = require('@angular/compiler-cli/linker').needsLinking(
+          this.resourcePath,
+          source,
+        );
+
+        // If the linker entry point is present then there is linker support
+        hasLinkerSupport = true;
+      } catch {
+        // Fallback for Angular versions less than 11.1.0
+        requiresLinking =
+          source.includes('ɵɵngDeclareDirective') || source.includes('ɵɵngDeclareComponent');
```

Comment:
```suggestion
        requiresLinking = false;
```

Given that there is no support for linking prior to 11.1.0